examples/fxa-client/src/devices-api.rs

```diff
@@ -4,7 +4,7 @@
 
 use cli_support::prompt::prompt_string;
 use dialoguer::Select;
-use fxa_client::{device, Config, FirefoxAccount, IncomingDeviceCommand};
+use fxa_client::internal::{device, Config, FirefoxAccount, IncomingDeviceCommand};
```

Comment:
I think it would be really nice if this example used the public interface as exposed by UniFFI, but the `Config` abstraction is not part of the UniFFI API. It could probably be added, but I've gotta stop somewhere...

Reply:
yeah - some tension I've found here is probably phrased better as "sometimes we want an example to be more like a diagnotic/test utility than an example of real-world usage" - ie, I explicitly *want* to call internal stuff in my example while not making it public". ie2, sometimes I want my examples to have crate visibility.